adb-export.sh

```
@@ -0,0 +1,2 @@
+adb reverse tcp:8080 tcp:8080
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)